pom.xml

```diff
@@ -241,6 +249,7 @@
     <ojdbc.version>21.3.0.0</ojdbc.version>
     <db2jcc4.version>4.21.29</db2jcc4.version>
     <webdrivermanager.version>3.1.1</webdrivermanager.version>
+    <logback-classic-db.version>1.2.11.1</logback-classic-db.version>
```

Comment:
We believe that logback-classic-db should be an exceptional property, since the numbering system is different here than in other places due to logback's various circumstances.

Reply:
For example, logback may be continuously released, but logback-db may not be, so I agree your opinion.
However, the property name prefix would be better for the project or group `logback-db` rather than the artifact `logback-classic-db`. Thanks.

Reply:
Yes, let's say logback-db.

Reply:
Fixed with 01cb6fd6091321d21e9ecd2e52918755fc0d26e6 in https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/986